packages/zk-certificates/contracts/mock/DevnetGuardian.sol

```diff
@@ -29,6 +29,9 @@ contract DevnetGuardian {
         bytes32 zkKYCRecordHash,
         bytes32[] memory merkleProof
     ) public {
+        recordRegistry.registerToQueue(
```

Comment:
I think it would make more sense to use the same interface in the DevNet guardian. Then any frontend or backend can use the Devnet Guardian to issue zkCerts in the same way as the Registry directly.

Reply:
Added a commit to fix that.